newIDE/app/src/ResourcesList/index.js

```diff
@@ -79,6 +83,38 @@ export default class ResourcesList extends React.Component<Props, State> {
     this.props.onDeleteResource(resource);
   };
 
+  _scanForNewResources = () => {
+    const resourcesList = this
+    const project = this.props.project
+    const resourcesManager = project.getResourcesManager()
+    console.log('Scanning the project folder for new resources...');
+    const projectPath = path.dirname(project.getProjectFile());
+
+    const getDirectories = (src, callback) => {
+      glob(src + '/**/*.png', callback);
```

Comment:
Should we only scan for `png` images? Should we include `jpg` too?

Reply:
I am against scanning for jpg files personally, because they have no support for an alpha channel. 
Jpegs have better compression though.

If you want to have good compression AND support for alpha channel - it is better to actually use webp files. Not sure if gdevelop.js has support for those though :}

Godot devs added support for them, even though they are not very popular

Reply:
While it's true that jpg should be avoided most of the time, they can be still be used for large backgrounds, so we should handle them properly. 
Pretty sure users will report a bug if they add a jpg to their folder and don't see it when using this feature.

Reply:
Added now :)
jpg, jpeg will be found